content/talks/2017-04-20-EC cards inside out.md

```diff
@@ -8,7 +8,7 @@ speakers:
 author: Matthias Beyer
 slides: false
 tags:
-    - make
+    - EC cards
```

Comment:
nope, I won't let that through as tag.

Can you use smth else here? Something more one-wordish? (lowercase pls)